media/js/base/core-datalayer.js

```diff
@@ -25,6 +25,33 @@ if (typeof Mozilla == 'undefined') {
         return pageId ? pageId : pathName.replace(/\/[^\/]*/, '');
     };
 
+    /** Returns whether page has download button.
+    * @param {String} path - URL path name fallback if page ID does not exist.
+    * @return {String} boolean.
+    */
+    Analytics.pageHasDownload = function() {
+        return ( $('[data-download-os]').length > 0 ) ? 'true' : 'false';
+    };
+
+    /** Returns whether page has video.
+    * @param {String} path - URL path name fallback if page ID does not exist.
+    * @return {String} boolean.
+    */
+    Analytics.pageHasVideo = function() {
+        return ( $('video').length > 0 ) ? 'true' : 'false';
+    };
+
+    /** Returns page version.
+    * @param {String} path - URL path name fallback if page ID does not exist.
+    * @return {String} version number from URL.
+    */
+    Analytics.getPageVersion = function(path) {
+        var pathName = path ? path : document.location.pathname;
+        var versionResults = /firefox\/(\d{1,2}\.\d{1,2}\.?\d{0,2})/.exec(pathName);
```

Comment:
This was not intentional. Will make the update to capture the Deve Edition and Nightly version numbers.